src/components/settings/settings.js

```diff
@@ -128,9 +128,6 @@ class Settings extends Component {
                 <FormattedMessage id="ui-eholdings.settings.accessStatusTypes" />
               </NavListItem>
             </IfPermission>
-            <NavListItem to={`/settings/eholdings/${configuration.id}/users`}>
```

Comment:
It looks like we have two problems:

1. We're missing `usergroups.collection.get` permission to receive user groups in our `ui-eholdings.settings.assignedUser`.
2. `ui-eholdings.settings.assignedUser` permission is not used in `settings-assigned-users` to hide the button if user doesn't have it.

Reply:
About second. True, but we hide the button in `settings`, is it not enough?

Reply:
From description in ticket I got that we need to show the users list even without this permission, we only need to hide the `Assign users` button

Reply:
Let's ask Khalilah what's the exact expected behavior and which permission should grant the ability to view assigned users

Reply:
Oh, I understand where is my mistake. By the way, in your opinion it is normal that I delete this component? We have the similar wrapped with permissions below.

Reply:
Well, if we need to show the assigned users to anyone then we don't need to use this permission for route